src/Microsoft.PowerShell.Commands.Utility/resources/WebCmdletStrings.resx

```diff
@@ -210,4 +210,7 @@
   <data name="JsonDeserializationFailed" xml:space="preserve">
     <value>Conversion from JSON failed with error: {0}</value>
   </data>
+  <data name="NotSuccessStatusCode" xml:space="preserve">
+    <value>The remote server returned an error: ({0:d}) {1}.</value>
```

Comment:
I believe "d" in String.Format is reserved for DateTime formatting.  I was using this as reference: http://www.csharp-examples.net/string-format-datetime/